rundeckapp/grails-app/services/rundeck/services/scm/ScmLoaderService.groovy

```diff
@@ -43,6 +42,7 @@ class ScmLoaderService implements EventBusAware {
     final Map<String, ScheduledFuture> scmProjectLoaderProcess = Collections.synchronizedMap([:])
     final Map<String, Boolean> scmProjectInitLoaded = Collections.synchronizedMap([:])
     final Map<String, ScmPluginConfigData> scmPluginMeta = Collections.synchronizedMap([:])
+    final Map<String, ScmPluginConfigData> scmFailedProjectInit = Collections.synchronizedMap([:])
```

Comment:
I think these changes to the ScmLoaderService should not be part of this PR, it doesn't have to do with the renamed job import, right?

Reply:
on